gnovm/docs/go-gno-compatibility.md

```diff
@@ -6,33 +6,57 @@
-| struct      | TBD    |
-| switch      | TBD    |
-| type        | TBD    |
-| var         | TBD    |
+| keyword     | support                |
+|-------------|------------------------|
+| break       | full                   |
+| case        | full                   |
+| const       | full                   |
+| continue    | full                    |
+| default     | full                   |
+| defer       | full                   |
+| else        | full                   |
+| fallthrough | full                   |
+| for         | full                   |
+| func        | full                   |
+| go          | missing (after launch) |
+| goto        | full                   |
+| if          | full                   |
+| import      | full                   |
+| interface   | full                    |
+| package     | full                   |
+| range       | full                    |
+| return      | full                   |
+| select      | TBD                    |
+| struct      | full                   |
+| switch      | full                   |
+| type        | full                   |
+| var         | full                   |
+
+## Native types
+
+| type                                          | usage                  | persistency                               | comment                                   |
+|-----------------------------------------------|------------------------|-------------------------------------------|-------------------------------------------|
+| `bool`                                        | full                   | full                                      |                                           |
+| `byte`                                        | full                   | full                                      |                                           |
+| `float32`, `float64`                          | full                   | full                                      | need review of determinism across systems |
+| `int`, `int8`, `int16`, `int32`, `int64`      | full                   | full                                      | potential support of 128, 256 later       |
+| `uint`, `uint8`, `uint16`, `uint32`, `uint64` | full                   | full                                      | potential support of 128, 256 later       |
+| `string`                                      | full                   | full                                      |                                           |
+| `rune`                                        | full                   | full                                      |                                           |
+| `interface{}`                                 | full                   | full                                      |                                           |
+| `[]T` (slices)                                | full                   | partial (depends on T)                    |                                           |
```

Comment:
I think full is sufficient here with asterisk since it depends on T. 
In terms of the logic for slice itself assuming supported T, it is supported, so I think that's what we should go by.